agent/agent-bootstrap/src/main/java/com/microsoft/applicationinsights/agent/bootstrap/diagnostics/status/StatusFile.java

```diff
@@ -255,6 +216,21 @@ public void run() {
         "StatusFileJsonWrite");
   }
 
+  private static boolean writable() {
+    if (!DiagnosticsHelper.useAppSvcRpIntegrationLogging()) {
+      return false;
+    }
+
+    File file;
+    if (DiagnosticsHelper.isOsWindows()) {
+      file = new File("/home");
+    } else {
+      file = new File("/var/log");
+    }
+
+    return file.canWrite();
```

Comment:
should we be checking canWrite on the log directory?
```suggestion
    return logDir.canWrite();
```